packages/core/src/system/Popup/styles/types.ts

```diff
@@ -0,0 +1,10 @@
+import { Axis } from '@core/utils';
+
+export type OmittedPopupStyleParams = Record<Axis, number>;
+
+export interface PopupStyleParams extends OmittedPopupStyleParams {
+    /** Полное отключение Backdrop компонента
```

Comment:
Порядок слов обратный не всегда необходим

![image](https://user-images.githubusercontent.com/54769688/213392211-0ff3543d-b6ef-4cdc-a6d7-7535f87af8aa.png)



```suggestion
    /** Полное отключение компонента Backdrop
```